pyproject.toml

```diff
@@ -38,7 +38,8 @@ extend-exclude = [
     "airflow/providers/google/ads/_vendor/*",
     # The files generated by stubgen aren't 100% valid syntax it turns out, and we don't ship them, so we can
     # ignore them in ruff
-    "airflow/providers/common/sql/*/*.pyi"
+    "airflow/providers/common/sql/*/*.pyi",
+    "airflow/migrations/versions/*.py"
```

Comment:
Following up on [this comment](https://github.com/apache/airflow/pull/31135#discussion_r1187983707). These migration files are generated automatically by alembic. I could not find a way to configure alembic to generate D400 compatible docstring. Since these files are automatically generated, I think it is fine to exclude them from this check